Sources/Sentry/SentryTracer.m

```diff
@@ -212,8 +225,15 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
     return self;
 }
 
+- (nullable SentryTracer *)tracer
+{
+    return self;
+}
```

Comment:
Oh, after thinking looking at it again, this is because `SentryTracer` is a subclass of `SentrySpan`, right? Can you explain when `-[SentryTracer tracer]` would be called? This is a code smell.

Reply:
As you mentioned, `SentryTracer` is a subclass of `SentrySpan`. 
Some places in the project contains a collection of SentrySpan that need to get the root of the transaction. All it needs to do is calling span.tracer